transformer/index.js

```diff
@@ -8,7 +8,7 @@ const { File } = require("babel-core/lib/transformation/file")
 const { SourceMapConsumer } = require("source-map")
```

Comment:
Might be worth moving to https://github.com/ds300/react-native-typescript-transformer - it is based on our Emission code

Reply:
👍